### PR TITLE
Update templates to redirect to DevCom

### PR DESCRIPTION
**Pull request by benmcmorran**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,17 @@
...
-
-- **discussion** – Denotes a discussion on the board that does not relate to a specific feature.
-  - Example: Visual C++ for Linux Development is not open source 
-
-Additional tags may be used to denote specific types of issues. 
+Bugs and new suggestions should be reported on the [Developer Community forum](https://developercommunity.visualstudio.com). It is centralized, better equipped for group voting, and has more visibility.
 
 ## Closing
-Issues may be closed by the original poster at any time. An issue can be closed if:
+New issues will be closed immediately. Bugs and suggestions should be reported on the [Developer Community forum](https://developercommunity.visualstudio.com).
```
Comment (itodirel) [inline README.md]: do we want to say that? "will", or "might";

if a customer gets us usable information as a bug, we should take in account and not dismiss it, maybe close and open a bug in VSO; if it's a question, we can say devcom

File: README.md [hunk 2]
```diff
@@ -1,23 +1,17 @@
 # Visual C++ for Linux Development
 
-This repo is for discussing existing or new features and for addressing questions related to the Visual C++ for Linux Development extension.
+This is a legacy repo that archives old discussion and questions related to the Visual C++ for Linux Development extension.
```
Comment (itodirel) [inline README.md]: I think we should mention also that existing issues will be honored and driven to completion

File: README.md (hunk 1 above)
Comment (benmcmorran) [reply README.md]: My concern if we soften this language is that we're not offering clear guidance on the appropriate way to report bugs. Why is it _sometimes_ okay to create an issue here? Closing the issue doesn't have to mean we don't use the feedback, but if we want to discourage new issue creation, I think we need to be clear about our approach.

Comment (itodirel) [reply README.md]: ok, how about then "New issues will be closed", without immediately?

**Pull request merged**